_includes/pbc/all/install-features/202307.0/install-the-service-points-customer-account-management-feature.md

```diff
@@ -152,6 +160,25 @@ class ShopApplicationDependencyProvider extends SprykerShopApplicationDependency
 }
 ```
 
+### 4) Set up FE part
```

Comment:
Add the introductory sentence; remove the header otherwise, as we can't have just headings without explanations.